lmdeploy/messages.py

```diff
@@ -140,6 +140,10 @@ class PytorchEngineConfig:
         tp (int): Tensor Parallelism. default 1.
         session_len (int): Max session length. Default None.
         max_batch_size (int): Max batch size. Default 128.
+        cache_max_entry_count (float): the percentage of gpu memory occupied
```

Comment:
@grimoire add the argument in the cli in this PR https://github.com/grimoire/lmdeploy/pull/1